pkg/collector/py/loader.go

```diff
@@ -292,13 +278,39 @@ func expvarPy3Warnings() interface{} {
 	return py3Warnings
 }
 
-func addExpvarPy3Warnings(checkName string, warnings []string) (float64, string) {
+func addExpvarPy3Warnings(checkName string, checkFilePath string) {
```

Comment:
this function does more than simply add the warnings to expvar, so I think we should:
1. change its name to reflect that (maybe to `reportPy3Warnings` for example)
2. add a comment on the function